### PR TITLE
Add authorization state to storages API endpoint

### PR DESCRIPTION
**Pull request by fraber**
Finished integration testing with live Nextcloud system:
- renamed :auth_failed into :failed_authorization
- fixed issue with refreshing tokens twice in case of errors

### CODE REVIEW

File: app/services/oauth_clients/connection_manager.rb [hunk 1]
```diff
@@ -31,7 +31,8 @@
 
 module OAuthClients
   class ConnectionManager
-    AUTHORIZATION_CHECK_PATH = '/ocs/v2.php/cloud/capabilities'.freeze
+    # Nextcloud API endpoint to check if Bearer token is valid
+    AUTHORIZATION_CHECK_PATH = '/ocs/v1.php/cloud/user'.freeze
```
Comment (Kharonus) [inline app/services/oauth_clients/connection_manager.rb:35]: is this the endpoint gotten from Julien? any idea, why it is a `v1` endpoint?

File: config/locales/en.yml [hunk 2]
```diff
@@ -3217,7 +3217,7 @@ en:
     urn_connection_status:
       connected: "Connected"
       error: "Error"
-      auth_failed: "Authentication failed"
+      failed_authorization: "Authentication failed"
```
Comment (Kharonus) [inline config/locales/en.yml]: Guess the localized text should also say: `Authorization failed`

**Review by Kharonus — COMMENTED** [review]
One small fix in texts required, code LGTM. 

But there are 2 rubocop issues to solve and at least 2 unit tests failing, which need fixing.

File: app/services/oauth_clients/connection_manager.rb (hunk 1 above)
Comment (fraber) [reply app/services/oauth_clients/connection_manager.rb:35]: Rrright, from Julien to Wieland to myself.
This is supposably the cheapest way to test for a valid Bearer token.

**Pull request merged**